python/ml4ir/base/model/relevance_model.py

```diff
@@ -234,16 +246,25 @@ def fit(
             monitor_mode=monitor_mode,
             monitor_metric=monitor_metric,
             patience=patience,
-            track_experiment=track_experiment,
         )
         if self.is_compiled:
-            self.model.fit(
+            history = self.model.fit(
                 x=dataset.train,
                 validation_data=dataset.validation,
                 epochs=num_epochs,
                 verbose=True,
                 callbacks=callbacks_list,
             )
+
+            # Write metrics for experiment tracking
+            train_metrics = dict()
+            for metric, value in history.history.items():
+                if not metric.startswith("val_"):
```

Comment:
Seems weird, can't we have `if metric.startswith("train_")`?

Reply:
This is not a check though. It's the key for the dictionary. How can I do `.startswith`?